source/documentation/runbooks/004-anlytical-platform-compute-maintenance.html.md.erb

```diff
@@ -0,0 +1,81 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Analytical Platform Compute Maintenance
+last_reviewed_on: 2024-10-28
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+On the first day of the month the workflow [schedule-issue-compute-infrastructure.yml] (https://github.com/ministryofjustice/analytical-platform/blob/bba07f4908b443a2c15bf94738c8b84c5211e9f9/.github/workflows/schedule-issue-compute-infrastructure.yml) will automatically raise a ticket for [Maintenance - Analytical Platform Compute ] (https://github.com/ministryofjustice/analytical-platform/issues/5661 )
+
+This maintenance ticket includes EKS cluster Upgrade if applicable and/or patching to ensure all components are as up to date as possible.
+
+##  Check for new release
+
+Check if a new release of Amazon EKS Kubernetes has been made available [here](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
+)
+
+If a new EKS release available carry out EKS upgrade and patch all other components
+
+Or
+
+If no EKS release available patch all other components including addons
+
```

Comment:
I'd alter the english here to have full stops or to be something like the below:
If 
a) 
b)